### DISCUSSION TITLE
API DOC issue - returns only one letter for the address state code.

### DISCUSSION THREAD

**Discussion by EnriqueCarreon08** [woocommerce/woocommerce#45554] [General]
When calling the order resource the shipping and billing address contains the **state** field with one letter ( B ) before this field returned
two letters ( BA = Buenos Aires ) for Argentina country.

```javascript
        "billing": {
            "first_name": "Maria",
            "last_name": "Prueba",
            "company": "",
            "address_1": "126",
            "address_2": "",
            "city": "Madariaga",
            "state": "B",
            "postcode": "7167",
            "country": "AR",
            "email": "kamerlingh.dev@gmail.com",
            "phone": ""
        },
        "shipping": {
            "first_name": "Maria",
            "last_name": "Prueba",
            "company": "",
            "address_1": "126",
            "address_2": "",
            "city": "Madariaga",
            "state": "B",
            "postcode": "7167",
            "country": "AR",
            "phone": ""
        }
``` 
Why this was changed and the API Documentation wasn't updated?

**Comment by coreymckrill** [comment]
@EnriqueCarreon08 It appears that we have used the single letter code for Argentinian states for several years. Here is where the data is that is used to populate the API response: https://github.com/woocommerce/woocommerce/blob/b13f0b0dcba06d9639a113f55d53012b18e92dd0/plugins/woocommerce/i18n/states.php#L57-L57
